lib/Remotery.h

```diff
@@ -86,10 +86,10 @@ documented just below this comment.
 // Compiler identification
 #if defined(_MSC_VER)
     #define RMT_COMPILER_MSVC
-#elif defined(__GNUC__)
-    #define RMT_COMPILER_GNUC
 #elif defined(__clang__)
     #define RMT_COMPILER_CLANG
+#elif defined(__GNUC__)
```

Comment:
I didn't know that was there! I'd just remove them all to be fair. They're not used and just cluttering up the API.